CMakeLists.txt

```diff
@@ -115,11 +115,14 @@ if(ENABLE_LTO)
 endif()
 
 if(ENABLE_OPENMP)
-    find_package( OpenMP REQUIRED )
-    include(FindOpenMP)
-    if(OPENMP_FOUND)
+    if(APPLE)
+        set(CMAKE_PREFIX_PATH "/usr/local/opt/libomp;${CMAKE_PREFIX_PATH}")
```

Comment:
(I may be talking totally stupid things)
Shouldn't `OpenMP_CXX_INCLUDE_DIR` be used as an _input_ to provide this path?
Described at the bottom of this page:
https://cmake.org/cmake/help/latest/module/FindOpenMP.html

Reply:
No, but it looks like we can use OpenMP_DIR instead, I will test that out.

See https://cmake.org/cmake/help/latest/command/find_package.html#id5

Reply:
Just tried with the `OpenMP_DIR` variable and that didn't work, looking into the docs some more, those (and also the HINTS and PATHS options to find_package) are for specifying the location for the FindOpenMP.cmake file, so I think my current solution is correct.
I will jump on IRC tomorrow and ask some people in the cmake channel.

Reply:
I don't think you need the #pragma omp single or the curly braces. The code should be running single threaded in there anyway. The #if of course is needed because that function won't exist in a non-OMP build.

Reply:
@ruevs Looks like I dismissed your answer to quickly, I found it at the bottom of that page now too =)

And adding
```
set(OpenMP_CXX_INCLUDE_DIR "/usr/local/opt/libomp/include")
set(OpenMP_C_INCLUDE_DIR "/usr/local/opt/libomp/include")
 ```
 Seems to be enough to make thing work correctly.